README.md

```diff
@@ -28,6 +28,45 @@ jobs:
 
 Info: You should copy not only step, but also another parts above (run only on labeled pull requests with label `dependencies`) to work it correctly.
 
+## Using aws-cli with proxy
+
+It is possible to configure the step that using aws-cli binary to make the connection through proxy. To do this, you must set the appropriate [environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-proxy.html).
+
+Detailed infromation about proxy you can find [here](https://secure.cldsrv.biz/f/87609).
```

Comment:
I'm not sure if we should expose this externally

Reply:
I've thought about it. Maybe it is actually better to remove this link.